docs/getting-started-guides/locally.md

```diff
@@ -129,6 +132,19 @@ cluster/kubectl.sh create -f docs/user-guide/pod.yaml
 
 Congratulations!
 
+### Hacking hack-local-up cluster to test more sophisticated deployments, such as HA.
+
+To test highly available apiservers, you can modify local-up-cluster.sh as follows.
+
+1) Write a kubernetes token file which has a kubelet user with a token defined.  Lets call it /tmp/mytokens for concreteness.
+
+2) Launch the apiserver with the `--token-auth-file=/tmp/mytokens` and `--apiserver-node-port=30279` options.  Of course 30279 is just an example port.
```

Comment:
Oh, gotcha. :)

On Wed, Sep 23, 2015 at 3:33 PM, Clayton Coleman <notifications@github.com>
wrote:

> In docs/getting-started-guides/locally.md
> <https://github.com/kubernetes/kubernetes/pull/13978#discussion_r40266673>
> :
>
> > @@ -129,6 +132,19 @@ cluster/kubectl.sh create -f docs/user-guide/pod.yaml
> >
> >  Congratulations!
> >
> > +### Hacking hack-local-up cluster to test more sophisticated deployments, such as HA.
> > +
> > +To test highly available apiservers, you can modify local-up-cluster.sh as follows.
> > +
> > +1) Write a kubernetes token file which has a kubelet user with a token defined.  Lets call it /tmp/mytokens for concreteness.
> > +
> > +2) Launch the apiserver with the `--token-auth-file=/tmp/mytokens` and `--apiserver-node-port=30279` options.  Of course 30279 is just an example port.
>
> I was referring to the service IP (the second paragraph in Tim's response)
> - so I'm saying the same thing you are. Node port is only interesting
> because of ingress.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/13978/files#r40266673>.
>